test/c3/appframework/appframework_test.go

```diff
@@ -2643,8 +2643,8 @@ var _ = Describe("c3appfw test", func() {
 		})
 	})
 
-	// CSPL-1647
-	XContext("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
+	// CSPL-1604
+	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
 		It("integration, c3, appframeworkc3, appframework: can deploy a C3 SVA and have apps installed and updated locally on Cluster Manager and Deployer for manual polling", func() {
```

Comment:
There are more tests that need to be enabled back, both in this file as well as `test/m4/appframework/appframework_test.go`

Reply:
Yup just enabled one in m4. Let me know if there are other tests that need to be enabled also.

Reply:
```
grep "CSPL-1647" -rn *                      
test/m4/appframework/appframework_test.go:1047:	// CSPL-1647
test/m4/appframework/appframework_test.go:1231:	// CSPL-1647
test/c3/appframework/appframework_test.go:1642:	// CSPL-1647
test/c3/appframework/appframework_test.go:1826:	// CSPL-1647
```